### PR TITLE
libgda: enable mysql, sqlcipher and web

### PR DESCRIPTION
**Pull request by raedrizqie**
- enable mysql, sqlcipher and web
- fix arch refs

### CODE REVIEW

File: mingw-w64-libgda/005-use-mysql-from-mariadb.patch [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
++++ libgda-6.0.0/meson.build	2021-12-21 13:42:39.324465066 -0600
+@@ -64,7 +64,7 @@
+ libxslt_dep = dependency('libxslt', required: false)
+ jsonglib_dep = dependency('json-glib-1.0',required : false)
+ postgres_dep = dependency ('libpq', required: false)
+-mysql_dep = dependency ('mysqlclient', required: false)
++mysql_dep = dependency ('libmariadb', required: false)
+ mysql_args = []
+ if mysql_dep.found ()
+ if mysql_dep.version ().version_compare ('>21.0.0')
```
Comment (MehdiChinoune) [inline mingw-w64-libgda/005-use-mysql-from-mariadb.patch:11]: I don't think this patch is required after #19808

Comment (raedrizqie) [reply mingw-w64-libgda/005-use-mysql-from-mariadb.patch:11]: nice! i will update the PR later

**Comment by MehdiChinoune** [comment]
Some svg files removed!

**Comment by raedrizqie** [comment]
hmm.. even just a rebuild in CI produced a package without all those svg files.. maybe something about the updated dependencies?

**Pull request closed without merging**